Aquarius/Src/Aquarius/Renderer/SpriteSheet.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "Aquarius/Renderer/Texture.h"
```

Comment:
Is this include necessary since you forward declare?

Reply:
Nope, will remove